kafka-rest/src/test/java/io/confluent/kafkarest/integration/v3/ProduceActionIntegrationTest.java

```diff
@@ -64,15 +64,14 @@
 import org.apache.kafka.common.header.internals.RecordHeader;
 import org.apache.kafka.common.serialization.ByteArrayDeserializer;
 import org.junit.Before;
-import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
 // TODO ddimitrov This continues being way too flaky.
 //  Until we fix it (KREST-1542), we should ignore it, as it might be hiding even worse errors.
-@Ignore
+// @Ignore
```

Comment:
I have mixed feelings about this - I totally want to enable it, but I think until we have figured out how to avoid SR calling `System.exit(1)` on us from time to time, it will be a pretty annoying problem for our CI results.

Reply:
I didn't mean to re-enable this, was just using the test locally to test the deserialization changes.